services/alert/service.go

```diff
@@ -763,17 +767,17 @@ func (s *Service) createHandlerFromSpec(spec HandlerSpec) (handler, error) {
 		keyvalue.KV("topic", spec.Topic),
 	}
 	switch spec.Kind {
-	case "aggregate":
-		c := newDefaultAggregateHandlerConfig(s.EventCollector)
```

Comment:
Er why was this removed?

Reply:
This was mistake, thanks for pointing that!